e2e test is broken

## Describe the bug

While reviewing https://github.com/woocommerce/storefront/pull/1833, I noticed that the e2e tests are failing. It turned out that the Storefront theme is no longer activated when running `npm run wp-env`. 

## To Reproduce

1. Check out the trunk.
2. Start Docker Desktop.
3. Open the terminal and run `npm run wp-env && npm run test:e2e`.
4. See that the e2e test is broken.

## Output

```
npm run test:e2e                                                                                                          ✔  took 5s   14.18.1  

> storefront@3.9.1 test:e2e /Users/nielslange/Themes/storefront
> jest

 FAIL  e2e/specs/browser.test.js
  Storefront
    ✕ should have "built with Storefront" footer (510 ms)

  ● Storefront › should have "built with Storefront" footer

    TimeoutError: Text not found "Built with Storefront & WooCommerce."

      waiting for function failed: timeout 500ms exceeded

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        1.328 s, estimated 4 s
Ran all test suites.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! storefront@3.9.1 test:e2e: `jest`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the storefront@3.9.1 test:e2e script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nielslange/.npm/_logs/2021-11-09T05_57_30_743Z-debug.log
```

## Expected behavior

The e2e test should succeed.

## Additional information

Via https://github.com/WordPress/gutenberg/issues/26766, I found out that it's no longer possible to activate a certain theme via `.wp-env.json`. To activate a theme, the following commands can be used:

```
// Activate Storefront on the WordPress development site
$ npm run wp-env run cli "wp theme activate storefront"

// Activate Storefront on the WordPress test site
$ npm run wp-env run tests-cli "wp theme activate storefront"
```

To ensure that Storefront is loaded by default, I propose to change `"test:e2e": "jest"` to `"test:e2e": "npm run wp-env run tests-cli "wp theme activate storefront" && jest"`.

--- a/package.json
+++ b/package.json
@@ -58,7 +58,7 @@
 		"lint:js-fix": "eslint assets/js --ext=js,jsx,ts,tsx --fix",
 		"lint:php": "composer run-script phpcs ./inc",
 		"wp-env": "wp-env",
-		"test:e2e": "jest"
+		"test:e2e": "npm run wp-env run tests-cli 'wp theme activate storefront' && jest"
 	},
 	"jest": {
 		"preset": "jest-puppeteer",

